### PR TITLE
standardize Kernel layer + contracts + crosschain protos.

### PR DESCRIPTION
**Pull request by samfrb**

### CODE REVIEW

File: protobuf/kernel.proto [hunk 1]
```diff
@@ -2,312 +2,201 @@ syntax = "proto3";
...
-    repeated TransactionTrace InlineTraces = 8;
-    repeated LogEvent Logs = 9;
-    int64 Elapsed = 10;
-    ExecutionStatus ExecutionStatus = 11;
-    TransactionExecutingStateSet StateSet = 12;
-    bytes DeferredTransaction = 13;
-    string ReadableReturnValue = 14;
+    Hash transaction_id = 1;
+    bytes return_value = 2;
+    string std_out = 3;
```
Comment (rosona) [inline protobuf/kernel.proto]: rename

File: protobuf/kernel.proto [hunk 2]
```diff
@@ -2,312 +2,201 @@ syntax = "proto3";
...
+    string std_err = 4;
```
Comment (rosona) [inline protobuf/kernel.proto]: rename

File: protobuf/kernel.proto [hunk 3]
```diff
@@ -2,312 +2,201 @@ syntax = "proto3";
...
-    bytes SignerPubkey = 9999;
-    bytes Signature = 10000;
+    int32 version = 1;
+    int32 chain_id = 2;
+    Hash previous_block_hash = 3;
+    Hash merkle_tree_root_of_transactions = 4;
+    Hash merkle_tree_root_of_world_state = 5;
+    bytes bloom = 6;
+    int64 height = 7;
+    repeated bytes block_extra_data = 8;
```
Comment (rosona) [inline protobuf/kernel.proto]: rename extra_data

File: protobuf/kernel.proto (hunk 1 above)
Comment (samfrb) [reply protobuf/kernel.proto]: this has been removed, cf. #1805

File: protobuf/kernel.proto (hunk 2 above)
Comment (samfrb) [reply protobuf/kernel.proto]: this has been rename to "Error", cf. #1805

**Pull request merged**